server/data/controllers/AuthController.js

```diff
@@ -0,0 +1,28 @@
+const services = require('../services');
+
+const capitalizeServiceString = service =>
+    `${service.charAt(0).toUpperCase()}${service.slice(1)}`;
+
+module.exports = {
+    login: (req, res) => {
```

Comment:
Since there's no explicit use of `this` value in this function, you should just write these `login(req, res) {`